jck/runtime.api/playlist.xml

```diff
@@ -379,13 +379,25 @@
 		<testCaseName>jck-runtime-api-java_applet</testCaseName>
 		<disables>
 			<disable>
-				<comment>Disabled on aix for backlog/issues/486, Problem Report 147661 for Mac. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
-				<platform>x86-64_mac|ppc64_aix</platform>
+				<comment>Disabled on aix for backlog/issues/486. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>ppc64_aix</platform>
 				<impl>ibm</impl>
 			</disable>
 			<disable>
-				<comment>Disabled on aix for backlog/issues/486, Problem Report 147661 for Mac. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
-				<platform>x86-64_mac|ppc64_aix</platform>
+				<comment>Disabled on aixv for backlog/issues/486 Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>ppc64_aix</platform>
+				<impl>openj9</impl>
+			</disable>
+			<disable>
+				<comment>Disabled on Mac for Problem Report 147661. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>x86-64_mac</platform>
+				<version>8</version>
+				<impl>ibm</impl>
+			</disable>
```

Comment:
To make this simple, please remove the ones with `<impl>ibm</impl>`. There is no impl=ibm for AIX or Mac. The only impl=ibm is zos JDK11 atm.

Reply:
Thanks updated. I will have a separate PR cleaning up rest of the disabled targets that have this redundant exclude.